test/src/tests-consent.js

```diff
@@ -3,19 +3,14 @@ import sinon from 'sinon';
 import { urls } from './config';
 import { apiKey, MPConfig, testMPID } from './config';
 
-var getEvent = Utils.getEvent,
+var findBatch = Utils.findBatch,
     mockServer;
 
 describe('Consent', function() {
     beforeEach(function() {
         mockServer = sinon.createFakeServer();
         mockServer.respondImmediately = true;
 
-        mockServer.respondWith(urls.eventsV2, [
```

Comment:
All instances of mocking urls.eventsV2 have been removed since this endpoint is gone